### DISCUSSION TITLE
Update French translation

### DISCUSSION THREAD

**Discussion by ofuro88** [aclap-dev/video-downloadhelper#560] [Language & Translation]
Here an updated translation for French version.

[fr.txt](https://github.com/aclap-dev/video-downloadhelper/files/15415370/fr.txt)

**Comment by paulrouget** [comment]
Merci :D

**Comment by sprtt1** [comment]
Here an updated translation for French version.
[fr.txt](https://github.com/aclap-dev/video-downloadhelper/files/15444294/fr.txt)
